src/Connection.php

```diff
@@ -30,6 +30,31 @@ class Connection extends \Dibi\Connection {
     const DATABASES_KEY = "databases";
 
     /**
+     * @var string
+     */
+    const MEASURE_TOTAL_TIME_KEY = "measureTotalTime";
+
+    /**
+     * @var bool
+     */
+    protected $measureTime = false;
```

Comment:
What about $debugTime or $queryTime, and and with postfix enabled, $queryTimeEnabled = false ?